setup.cfg

```diff
@@ -0,0 +1,7 @@
+[flake8]
```

Comment:
This is in the root directory? Does it belong under the python directory?

Reply:
There's one in the python directory already, I vaguely remember there being a reason for adding this but let me try removing it and report back.